ddtrace/internal/datadog/profiling/dd_wrapper/src/crashtracker.cpp

```diff
@@ -105,15 +105,22 @@ Datadog::Crashtracker::set_tag(std::string_view key, std::string_view value)
 bool
 Datadog::Crashtracker::set_receiver_binary_path(std::string_view _path)
 {
+    static bool already_warned = false; // cppcheck-suppress threadsafety-threadsafety
```

Comment:
question: this function local static is not thread local, but "global"? I just never thought about this, but that is most likely why the comment is there.

Reply:
Yup, that's right.  These have global view.